### ISSUE TITLE
Proposal: KeyboardAcceleratorPlacementMode should default to Hidden

### ISSUE THREAD

**Issue by adrientetar** [microsoft/microsoft-ui-xaml#1134]
# Proposal: KeyboardAcceleratorPlacementMode should default to Hidden

When I add KeyboardAccelerators to my Page control, I get one of the accelerators displayed in a tooltip when hovering the page[\*]. That doesn't make much sense, and I had to go find KeyboardAcceleratorPlacementMode in the docs to disable it. Afaict, it only makes sense to default enabled for clickable items like Buttons (actually, #389 will be even better), so this proposal is to disable KeyboardAcceleratorPlacementMode globally and enable it in ButtonBase instead.

[\*] It probably could make sense to disable the automatic tooltip when there are more than one accelerator defined?

## Rationale

* The auto-generated tooltip only makes sense in clickable surfaces, where the click shortcut ought to be discoverable

## Scope

| Capability | Priority |
| :---------- | :------- |
| KeyboardAcceleratorPlacementMode defaults to Hidden globally | Must |
| defaults to Auto for ButtonBase | Should |

**Comment by jevansaks** [comment]
@Austin-Lamb what is the recommendation for how to set up page-level KeyboardAccelerators like this?

@adrientetar could you provide the snippet of code for how you specified the accelerator? I'm wondering if this is more of a bug than a spec issue.

**Comment by adrientetar** [comment]
@jevansaks, I just wrote my accelerators in `<Page.KeyboardAccelerators>` in the Page XAML.

**Comment by adrientetar** [comment]
> I'm wondering if this is more of a bug than a spec issue.

@jevansaks, why would this not be a spec issue, do you mean that the spec already includes the behavior that I suggest in the OP? cc: @marb2000

**Comment by jevansaks** [comment]
Right, I mean that I'm not actually sure what the "expected" behavior is here, so I defer to @austin-lamb and @marb2000.
